templates/listing-page-filters.twig

```diff
@@ -14,4 +14,17 @@
             </option>
         {% endfor %}
     </select>
+    <select class="form-select listing-page-select" id="post-type">
+        <option value="">{{__( 'All posts', 'planet4-master-theme' ) }}</option>
+        {% for key,post_type in post_types %}
+            {% set isSelected = current_post_type == post_type.slug %}
```

Comment:
What do you think to change to 
```suggestion
            {% set isSelected = current_post_type === post_type.slug %}
```

Reply:
We usually use `==` in twig templates, I see only in the google tag manager template we use `===` in one place 🤔